great_expectations/alias_types.py

```diff
@@ -0,0 +1,9 @@
+"""This module contains shared TypeAliases"""
```

Comment:
used `alias_types` instead of `types` to avoid weirdness with standard library module names, and to prevent this from being polluted with random types + class definitions.